source/frontend/src/features/properties/selector/utils.ts

```diff
@@ -1,7 +1,9 @@
 import { IMapProperty } from 'features/properties/selector/models';
 import { compact } from 'lodash';
 import { Api_PropertyFile } from 'models/api/PropertyFile';
-import { formatApiAddress, pidFormatter } from 'utils';
+import { pidFormatter } from 'utils';
+
+import { formatApiSummaryAddress } from './../../../utils/propertyUtils';
```

Comment:
Nit : we should get a root folder character and go from there instead of relative navigation for things like this

Reply:
agreed, my auto importer likely did this, will correct.